include/boost/histogram/utility/clopper_pearson_interval.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_UTILITY_CLOPPER_PEARSON_INTERVAL_HPP
+#define BOOST_HISTOGRAM_UTILITY_CLOPPER_PEARSON_INTERVAL_HPP
+
+#include <boost/histogram/utility/binomial_proportion_interval.hpp>
+#include <cmath>
+#include <utility>
+
+namespace boost {
+namespace histogram {
+namespace utility {
+
+template <class ValueType>
+class clopper_pearson_interval : public binomial_proportion_interval<ValueType> {
+  using base_t = binomial_proportion_interval<ValueType>;
+
+public:
+  using value_type = typename base_t::value_type;
+  using interval_type = typename base_t::interval_type;
+
+  explicit clopper_pearson_interval(deviation d = deviation{1.0}) noexcept
```

Comment:
You should accept a `confidence_level` here instead of a `deviation`. Since `deviation` and `confidence_level` are programmed to be convertible into each other, you can chose where which suits you best / requires less conversion in your implementation below in `operator()`. The Clopper-Pearson interval naturally needs the CL to compute alpha.